console/src/utils/migration/utils.ts

```diff
@@ -0,0 +1,283 @@
+  if (nullable) {
+    // ALTER TABLE <table> ALTER COLUMN <column> DROP NOT NULL;
+    const nullableUpQuery = `${ALTER_COL} DROP NOT NULL;`;
+    const nullableDownQuery = `${ALTER_COL} SET NOT NULL;`;
+    // check with original null
+    if (originalColNullable !== 'YES') {
+      migration.add(
+        getRunSqlQuery(nullableUpQuery),
+        getRunSqlQuery(nullableDownQuery)
+      );
+    }
+  } else {
+    // ALTER TABLE <table> ALTER COLUMN <column> SET NOT NULL;
+    const nullableUpQuery = `${ALTER_COL} SET NOT NULL;`;
+    const nullableDownQuery = `${ALTER_COL} DROP NOT NULL;`;
+    // check with original null
+    if (originalColNullable !== 'NO') {
+      migration.add(
+        getRunSqlQuery(nullableUpQuery),
+        getRunSqlQuery(nullableDownQuery)
+      );
+    }
+  }
+
+  /* column unique up/down migration */
+  if (unique) {
+    const uniqueUpQuery = `${ALTER_TABLE} ADD CONSTRAINT "${tableName}_${colName}_key" UNIQUE ("${colName}")`;
+    const uniqueDownQuery = `${ALTER_TABLE} DROP CONSTRAINT "${tableName}_${colName}_key"`;
+    // check with original unique
+    if (!originalColUnique) {
+      migration.add(
+        getRunSqlQuery(uniqueUpQuery),
+        getRunSqlQuery(uniqueDownQuery)
+      );
+    }
+  } else {
+    const uniqueDownQuery = `${ALTER_TABLE} ADD CONSTRAINT "${tableName}_${colName}_key" UNIQUE ("${colName}")`;
+    const uniqueUpQuery = `${ALTER_TABLE} DROP CONSTRAINT "${tableName}_${colName}_key"`;
+    // check with original unique
+    if (originalColUnique) {
+      migration.add(
+        getRunSqlQuery(uniqueUpQuery),
+        getRunSqlQuery(uniqueDownQuery)
+      );
+    }
+  }
+
+  /* column comment up/down migration */
+  const columnCommentUpQuery = `COMMENT ON COLUMN "${currentSchema}"."${tableName}"."${colName}" IS ${sqlEscapeText(
+    comment
+  )}`;
+
+  const columnCommentDownQuery = `COMMENT ON COLUMN "${currentSchema}"."${tableName}"."${colName}" IS ${sqlEscapeText(
+    originalColComment
+  )}`;
+
+  // check if comment is unchanged and then do an update. if not skip
+  if (originalColComment !== comment) {
+    migration.add(
+      getRunSqlQuery(columnCommentUpQuery),
+      getRunSqlQuery(columnCommentDownQuery)
+    );
+  }
+
+  /* rename column */
+  if (newName && colName !== newName) {
+    if (!gqlPattern.test(newName)) {
+      onInvalidGqlColName();
+    }
+    migration.add(
+      getRunSqlQuery(
+        `${ALTER_TABLE} rename column "${colName}" to "${newName}";`
+      ),
+      getRunSqlQuery(
+        `${ALTER_TABLE} rename column "${newName}" to "${colName}";`
+      )
+    );
+  }
+  const migrationName = `alter_table_${currentSchema}_${tableName}_alter_column_${colName}`;
+  return {
+    migrationName,
+    upMigration: migration.upMigration,
+    downMigration: migration.downMigration,
+  };
+};
+export const getDownQueryComment = (upqueries: RunSqlType[]) => {
+  if (upqueries instanceof Array && upqueries.length >= 0) {
```

Comment:
You provided typing for it being an array so either this check is redundant or type should say that it's an array or something else. 

If the second option, then let's use `Array.isArray`.